coalib/settings/Section.py

```diff
@@ -65,11 +66,17 @@ def bear_dirs(self):
         bear_dirs = path_list(self.get("bear_dirs", ""))
```

Comment:
Shouldnt we be using glob_list here ?
Same logic as glob_list for `section.get('files', "")` and so on above ?